osi_lane.proto

```diff
@@ -274,7 +296,7 @@ message Lane
         // check_if this.type is_different_to 4 else do_check is_set
         // \endrules
         //
-        repeated Identifier left_adjacent_lane_id = 5;
+        repeated Identifier left_adjacent_lane_id = 8;
```

Comment:
For backward-compatibility reasons please avoid renumbering. Instead, new fields should get not yet used numbers, while the textual order inside the file doesn't matter. (Applies also to changes below, of course)